### PR TITLE
Prepare 1.3.1 for release

### PR DESCRIPTION
**Pull request by t-eckert**
### Changes proposed in this PR ###  
- Updates the changelog
- Preps for the release 1.3.1

### CODE REVIEW

File: charts/consul/Chart.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 apiVersion: v2
 name: consul
-version: 1.3.1-dev
-appVersion: 1.17-dev
+version: 1.3.1
+appVersion: 1.16.4
```
Comment (wilkermichael) [inline charts/consul/Chart.yaml:7]: I think this should be `1.17.1`

File: charts/consul/Chart.yaml [hunk 2]
```diff
@@ -13,14 +13,14 @@ sources:
   - https://github.com/hashicorp/consul
   - https://github.com/hashicorp/consul-k8s
 annotations:
-  artifacthub.io/prerelease: true
+  artifacthub.io/prerelease: false
   artifacthub.io/images: |
     - name: consul
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17-dev
+      image: hashicorp/consul:1.16.4
```
Comment (wilkermichael) [inline charts/consul/Chart.yaml:19]: same should be 1.17.1.

Update your environment variables and run `prepare-release` again

File: charts/consul/values.yaml [hunk 3]
```diff
@@ -66,7 +66,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17-dev
+  image: hashicorp/consul:1.16.4
```
Comment (wilkermichael) [inline charts/consul/values.yaml:69]: should be `1.17.1`

**Review by wilkermichael — CHANGES_REQUESTED** [review]
I think you're using the wrong Consul version

**Comment by t-eckert** [comment]
@wilkermichael All fixed!

**Review by wilkermichael — APPROVED** [review]

**Pull request merged**